### PR TITLE
stm32/rcc: add i2s pll on some f4 micros

### PR DESCRIPTION
**Pull request by xoviat**
Adds the i2s pll on some f4 micros.

### CODE REVIEW

File: embassy-stm32/src/rcc/f4.rs [hunk 1]
```diff
@@ -28,11 +28,71 @@ pub struct Config {
...
     pub pclk1: Option<Hertz>,
     pub pclk2: Option<Hertz>,
+    pub plli2s: Option<Hertz>,
 
     pub pll48: bool,
 }
 
-unsafe fn setup_pll(pllsrcclk: u32, use_hse: bool, pllsysclk: Option<u32>, pll48clk: bool) -> PllResults {
+#[cfg(any(
+    feature = "stm32f411",
```
Comment (xoviat) [inline embassy-stm32/src/rcc/f4.rs:37]: Is there a way to select all of the chip variants without adding them all here? Something like a regex?

Comment (Dirbaio) [reply embassy-stm32/src/rcc/f4.rs:37]: build.rs defines some cfgs with cut down versions of the chip name https://github.com/embassy-rs/embassy/blob/master/embassy-stm32/build.rs#L837

so this works:

```rust
#[cfg(any(
    stm32f411,
    stm32f412,
    stm32f413,
    stm32f423,
    stm32f446,
))]
```

File: embassy-stm32/src/rcc/f4.rs [hunk 2]
```diff
@@ -28,11 +28,65 @@ pub struct Config {
     pub sys_ck: Option<Hertz>,
     pub pclk1: Option<Hertz>,
     pub pclk2: Option<Hertz>,
+    pub plli2s: Option<Hertz>,
```
Comment (Dirbaio) [inline embassy-stm32/src/rcc/f4.rs:31]: In chips without i2s pll, this field is still present, but does nothing. That might be confusing for the user. Could you change it to put the `cfg` in the field?

**Review by Dirbaio — APPROVED** [review]
thanks!

bors r+

**Pull request merged**